packages/components/src/components/multi-select/multi-select/multi-select-utils.ts

```diff
@@ -185,6 +188,7 @@ export const updateHighlightedOption = (
   direction: SelectDropdownDirectionInternal
 ): void => {
   const newIndex = getNewOptionIndex(options, direction);
+  // @ts-expect-error: Type number | void is not assignable to parameter of type number
   setNextOptionHighlighted(host, options, newIndex);
```

Comment:
```suggestion
  newIndex && setNextOptionHighlighted(host, options, newIndex);
```

Instead of calling the function with a undefined `newIndex`, we should only call it if a newIndex is found

Reply:
True, but the suggestion doesn't work as `newIndex === 0` equals `false`